supertokens_python/framework/flask/flask_response.py

```diff
@@ -0,0 +1,65 @@
+from werkzeug.http import dump_cookie
+
+from supertokens_python.framework.response import BaseResponse
+
+
+class FlaskResponse(BaseResponse):
+
+    def __init__(self, response=None):
+        super().__init__({})
+        self.response = response
+        self.headers = list()
```

Comment:
why not use dict here? It will always be a key value pair. also when searching for a value, you won't require to loop through the list

Reply:
If I remember well, the headers in the middleware is a list so I have to pass a list